client/manager.go

```diff
@@ -40,8 +40,7 @@ type Manager[T any] struct {
 func NewManager[T any](nc *nats.Conn,
 	construct func(nc *nats.Conn, config T) Client) *Manager[T] {
 	var x T
-	nodeType := reflect.TypeOf(x).Name()
-	nodeType = strings.ToLower(nodeType[0:1]) + nodeType[1:]
+	nodeType := data.ToCamelCase(reflect.TypeOf(x).Name())
```

Comment:
Is the `data` package the best place for this?

Reply:
that is fine with me -- a lot of people claim you should not have a data package, but I'm not there yet ...